slack/handler.go

```diff
@@ -102,15 +102,37 @@ func (r *rtmBot) handleMessage(ev *slack.MessageEvent) error {
 		return nil
 	}
 
-	if !strings.Contains(ev.Text, r.rtm.GetInfo().User.ID) {
+	if ev.BotID != "" {
 		return nil
 	}
 
-	if !strings.Contains(ev.Text, "nominate") {
+	if !strings.Contains(ev.Text, r.rtm.GetInfo().User.ID) {
 		return nil
 	}
 
-	return r.handleNominate(ev.User)
+	if strings.Contains(ev.Text, "nominate") {
```

Comment:
Maybe we can use a regexp to extract the "action", and then use it on a switch/case structure ?

Reply:
Good idea ! I will do it in a next PR.